### PR TITLE
chore(setup-documentation) setup swagger documentation

### PR DESCRIPTION
**Pull request by lemoissonM**
#### What does this PR do?
Setup swagger documentation with jsdoc
#### Description of Task to be completed?
add swagger-ui-express and swagger-jsdoc packages
add swagger definition
add documentation routes
#### How should this be manually tested?
-clone the repos
- run npm install
- run npm start dev
- in a browser navigate to the endpoint: /api-docs.
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/168781800
#### Questions:

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,93 +1,16 @@
...
-    path = require("path"),
-    methods = require("methods"),
-    express = require("express"),
-    bodyParser = require("body-parser"),
-    session = require("express-session"),
-    cors = require("cors"),
-    passport = require("passport"),
-    errorhandler = require("errorhandler"),
-    mongoose = require("mongoose");
+import express from 'express';
```
[the remark this answers is not in the document]
Comment (urbanishimwe) [reply src/index.js:1]: Lemoisson can you fix this issue, in your hound configuration

File: src/utils/swaggerDefinition.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import swaggerJsdoc from 'swagger-jsdoc';
```
[the remark this answers is not in the document]
Comment (urbanishimwe) [reply src/utils/swaggerDefinition.js:1]: you can add `eofline: 0` in `eslint.json` this error will be fixed

**Comment by urbanishimwe** [comment]
@lemoissonM  consider fixing those conflicts, you did well by de way

**Review by swaibat — APPROVED** [review]

**Review by alexismajya — APPROVED** [review]

File: src/index.js [hunk 3]
```diff
@@ -1,32 +1,18 @@
...
-app.use(cors());
-
-// Normal express config defaults
-app.use(require('morgan')('dev'));
+const app = express();
 
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.json());
 
-app.use(require('method-override')());
```
Comment (urbanishimwe) [inline src/index.js:15]: nice work here. however, remove these libraries from `package.json` also e.g `method-override` `morgan` etc.. it will help us all

Comment (lemoissonM) [reply src/index.js:15]: great! let me do it!

**Review by nakiwuge — APPROVED** [review]

**Pull request merged**